firefox-ios/Providers/Pocket/PocketProvider.swift

```diff
@@ -150,7 +156,7 @@ class PocketProvider: PocketStoriesProviding, FeatureFlaggable, URLCaching {
         return featureFlags.isCoreFeatureEnabled(.useMockData) && pocketKey.isEmpty
     }
 
-    private func getMockDataFeed(count: Int = 2, completion: (StoryResult) -> Void) {
+    private func getMockDataFeedold(count: Int = 2, completion: (StoryResult) -> Void) {
```

Comment:
We can remove this function if it's old and unused?

Reply:
I missed this during my cleanup, thanks :)